backend/cms/templates/pages/page.html

```diff
@@ -22,195 +22,182 @@ <h2 class="heading font-normal">
+				        </div>
+			        </div>
+			        {% if perms.cms.grant_page_permissions %}
+				        <span class="block font-bold mb-2">{% trans 'Additional permissions for this page' %}</span>
+				        <p class="italic">{% trans "This affects only users, who don't have the permission to change arbitrary pages anyway." %}</p>
+				        <div id="page_permission_table">
+					        {% include "pages/_page_permission_table.html" %}
+				        </div>
+			        {% endif %}
+			        <div class="pt-2 pb-4">
+				        <span class="block font-bold mb-4">{% trans 'Icon' %}</span>
+				        {% render_field page_form.icon id="icon" class="image-field" %}
+				        <label for="icon" class="font-bold bg-blue hover:bg-blue-dark focus:bg-blue-dark text-white font-bold py-3 pl-10 pr-4 rounded cursor-pointer relative">
+					        <i data-feather="upload"></i>
+					        <span class="standard_text">{% trans 'Set icon' %}</span>
+					        <span class="filename"></span>
+				        </label>
+			        </div>
+			        {% if page %}
 				        <div class="pt-2 pb-4">
-			                <span class="block font-bold mb-4">{% trans 'Delete page' %}</span>
-					        {% if page.children.all %}
-						        {% trans 'You cannot delete a page which has children.' %}
+					        <span class="block font-bold mb-4">{% trans 'Archive page' %}</span>
+					        {% if page.archived %}
+						        {% trans 'Page is archived.' %}
 					        {% else %}
-				                <button class="bg-red hover:bg-red-dark text-white font-bold py-2 px-4 rounded" onclick="confirmation_popup(event, '#confirm_delete_page_{{ page.id }}')">
-					                {% trans 'Delete this page' %}
-				                </button>
+						        <button class="bg-blue hover:bg-blue-dark text-white font-bold py-2 px-4 rounded" onclick="confirmation_popup(event, '#confirm_archive_page_{{ page.id }}')">
+							        {% trans 'Archive this page' %}
+						        </button>
 					        {% endif %}
-		                </div>
-		            {% endif %}
-	            {% endif %}
-            </div>
+				        </div>
+				        {% if user.is_superuser or user.is_staff %}
+					        <div class="pt-2 pb-4">
+						        <span class="block font-bold mb-4">{% trans 'Delete page' %}</span>
+						        {% if page.children.all %}
+							        {% trans 'You cannot delete a page which has children.' %}
+						        {% else %}
+							        <button class="bg-red hover:bg-red-dark text-white font-bold py-2 px-4 rounded" onclick="confirmation_popup(event, '#confirm_delete_page_{{ page.id }}')">
+								        {% trans 'Delete this page' %}
+							        </button>
+						        {% endif %}
+					        </div>
+				        {% endif %}
+			        {% endif %}
+		        </div>
+		        </div>
+	        </div>
         </div>
     </div>
 </form>
```

Comment:
In the beginning this file uses spaces for indentation, later tabs. I think we should use spaces everywhere because we're dong this in the Python code by default. Or is there a good reason to use tabs?

Reply:
Thanks for pointing that out! Yes, I think mixing them both makes no sense.